R/minimize.R

```diff
@@ -35,6 +35,116 @@ minimize <- function(
 
     args <- c(as.list(environment()), list(...))
 
+
+    if(is(initial_design, "OneStageDesign")) {
+        res <- do.call(.minimize_os, args = args)
```

Comment:
we can keep it, but looks like we might need to make minimize a method (which id like to avoid)